digitNbinTransfer/sentry.py

```diff
@@ -0,0 +1,20 @@
+from os import environ
+from sentry_sdk import set_user, init, add_breadcrumb
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+from sentry_sdk.utils import BadDsn
+
+def init_sentry(turn_on: bool = True, exit_on_fail=False) -> None:
+    try:
+        init(
+            dsn=environ['SENTRY_DSN'],
+            integrations=[AwsLambdaIntegration()],
+            traces_sample_rate=0.01,
+            max_breadcrumbs=50,
+        )
+    except BadDsn:
+        print('bad DSN. Sentry off.')
```

Comment:
good fallback logic. i like the exit_on_fail flag

Reply:
TY! I copy and pasted this from our CDK framework!!